packages/block-library/src/navigation/index.php

```diff
@@ -1620,7 +1620,7 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
  * Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.5
  * that are not present in Gutenberg's WP 6.5 compatibility layer.
  */
-if ( function_exists( 'set_ignored_hooked_blocks_metadata' ) && ! has_filter( 'rest_pre_insert_wp_navigation', $rest_insert_wp_navigation_core_callback ) ) {
+if ( function_exists( 'set_ignored_hooked_blocks_metadata' ) && ( ! has_filter( 'rest_pre_insert_wp_navigation', $rest_insert_wp_navigation_core_callback ) || ! has_filter( 'rest_pre_insert_wp_navigation', 'update_ignored_hooked_blocks_postmeta' ) ) ) {
```

Comment:
Nitpick, I think we should wrap this line 😬 

```suggestion
if (
	function_exists( 'set_ignored_hooked_blocks_metadata' ) && (
		! has_filter( 'rest_pre_insert_wp_navigation', $rest_insert_wp_navigation_core_callback ) ||
		! has_filter( 'rest_pre_insert_wp_navigation', 'update_ignored_hooked_blocks_postmeta' )
	)
) {
```